DllImportGenerator/DllImportGenerator.UnitTests/CodeSnippets.cs

```diff
@@ -446,5 +460,285 @@ partial class Test
 }}";
 
         public static string ArrayParameterWithNestedMarshalInfo<T>(UnmanagedType nestedMarshalType) => ArrayParameterWithNestedMarshalInfo(typeof(T).ToString(), nestedMarshalType);
+        public static string CustomStructMarshallingParametersAndModifiers = BasicParametersAndModifiers("S") + @"
+[NativeMarshalling(typeof(Native))]
+struct S
+{
+    public bool b;
+}
+
+struct Native
```

Comment:
It would be nice to try to establish naming convention for these and use it here. `Native` is very generic and it does not scale to have more than one in the given namespace.

Reply:
I agree that we need to establish a naming convention for these types, but I don't think we need to do that immediately. We'll need to do it when we decide we want to support generating the native types for structs.